src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/GCPBillingProjectWizard.test.ts

```diff
@@ -70,14 +77,34 @@ const testStep2HaveBillingChecked = () => {
   verifyUnchecked(getStep2BillingAccountNoAccessButton())
 }
 
+/* FIXME: this isn't actually testing state - it relies on which version has been rendered
+         the only reason the AddTerraAsBillingAccountUserStep is rendered instead of ContactBillingAccountAdministrator
+         is a logic quirk of the original implementation:
+          (isActive || isDone) &&
+         (!accessToBillingAccount
+           || (accessToAddBillingAccountUser !== undefined && !accessToAddBillingAccountUser)
+           ) ? contactBillingAccountAdministrator : addTerraAsBillingAccountUser
+    so at least for now, I'm just changing the disabled/unchecked checks to treat null elements as acceptable
+     */
 const testStep3InitialState = () => {
```

Comment:
This is the only spot where I materially changed the tests. As noted in the comments, the original test was dependent on components being rendered for a stage that we haven't gotten to yet, and the logic to determine which one was rendered was somewhat complex and noisy.